x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/other_mappings.json

```diff
@@ -0,0 +1,337 @@
+            "ignore_above": 1024,
+            "type": "keyword"
+          },
+          "location": {
+            "type": "geo_point"
+          },
+          "name": {
+            "ignore_above": 1024,
+            "type": "keyword"
+          },
+          "region_iso_code": {
+            "ignore_above": 1024,
+            "type": "keyword"
+          },
+          "region_name": {
+            "ignore_above": 1024,
+            "type": "keyword"
+          }
+        }
+      },
+      "hash": {
+        "properties": {
+          "md5": {
+            "ignore_above": 1024,
+            "type": "keyword"
+          },
+          "sha1": {
+            "ignore_above": 1024,
+            "type": "keyword"
+          },
+          "sha256": {
+            "ignore_above": 1024,
+            "type": "keyword"
+          },
+          "sha512": {
+            "ignore_above": 1024,
+            "type": "keyword"
+          }
+        }
+      },
+      "interface": {
+        "properties": {
+          "alias": {
+            "ignore_above": 1024,
+            "type": "keyword"
+          },
+          "id": {
+            "ignore_above": 1024,
+            "type": "keyword"
+          },
+          "name": {
+            "ignore_above": 1024,
+            "type": "keyword"
+          }
+        }
+      },
+      "os": {
+        "properties": {
+          "family": {
+            "ignore_above": 1024,
+            "type": "keyword"
+          },
+          "full": {
+            "fields": {
+              "text": {
+                "norms": false,
+                "type": "text"
+              }
+            },
+            "ignore_above": 1024,
+            "type": "keyword"
+          },
+          "kernel": {
+            "ignore_above": 1024,
+            "type": "keyword"
+          },
+          "name": {
+            "fields": {
+              "text": {
+                "norms": false,
+                "type": "text"
+              }
+            },
+            "ignore_above": 1024,
+            "type": "keyword"
+          },
+          "platform": {
+            "ignore_above": 1024,
+            "type": "keyword"
+          },
+          "version": {
+            "ignore_above": 1024,
+            "type": "keyword"
+          }
+        }
+      },
+      "pe": {
+        "properties": {
+          "company": {
+            "ignore_above": 1024,
+            "type": "keyword"
+          },
+          "description": {
+            "ignore_above": 1024,
+            "type": "keyword"
+          },
+          "file_version": {
+            "ignore_above": 1024,
+            "type": "keyword"
+          },
+          "original_file_name": {
+            "ignore_above": 1024,
+            "type": "keyword"
+          },
+          "product": {
+            "ignore_above": 1024,
+            "type": "keyword"
+          }
+        }
+      },
+      "threat": {
+        "properties": {
+          "indicator": {
```

Comment:
How are the experimental fields included here chosen? It looks like there are a couple fieldsets in the [experimental generated JSON](https://github.com/elastic/ecs/blob/master/experimental/generated/elasticsearch/component/threat.json) for threat.indicator  that don't show up here, notably `threat.indicator.file.*`, `.hash.*`, and `.registry.*` - are those purposely left out?

Reply:
CTI has, to date, only been pulling in what's needed in order to minimize the likelihood of incompatible mappings in the case where the experimental fields change. However, thank you for calling this out as we really need to formalize this process, or at the very least get product's opinion here.

@MikePaquette As part of 7.13 we have already added the [`event` fieldset](https://github.com/elastic/kibana/pull/95697/files) to both the alert mappings and the enrichment logic. However, as Marshall identified, the current indicator mappings are missing the nested `file`, `hash`, and `registry` fieldsets as currently specified in the RFC. Should we:

1. Do nothing
2. Add these mappings but leave them off of enrichment
3. Add both the mappings and the corresponding enrichment

Reply:
Sounds good - I went ahead and :+1:'d since any of the 3 approaches sound reasonable to me and the code changes look good.

Reply:
Confirmed the above needs/rationale with @MikePaquette and @peasead. Merging for now!